### ISSUE TITLE
Homepage banner papercuts

### ISSUE THREAD

**Issue by bertob** [flathub-infra/website#2512]
Top banner:
- [x] The vertical spacing between top bar and banners is a bit too high, I'd reduce it by a third or so
- [x] I'd reduce vertical spacing between app icon and app name on the top banner by about 5px

Set Up Flathub card:
- [x] The "App of the Day" and "Set up Flathub" cards should have the same height
- [x] I'd rephrase the text on the Set up Flathub card a bit, since it's no longer near the top I think it needs a bit more context. Maybe mentioning Flathub in the title (e.g. "Flathub is the App store for Linux" would already by enough?)
- [ ] The card background is a bit too high contrast, I think on the mockup I'd layered a slight white-to-transparent gradient over it
- [x] There's a bit too much vertical padding above the button, I'd reduce it by about half

![Screen Shot 2024-01-27 at 01 24 15](https://github.com/flathub-infra/website/assets/1908896/b7481515-0a1b-406d-ae1c-15d9b859fd6c)

Mobile:
- [x] It's a bit awkward that the icon in the top banner is smaller than the icon for app of the day. Could we increase the top banner height to make sure they are always the same size?
- [x] Vertical spacing between the banners feels excessive in this case, I'd use the same spacing as between the individual app cards

![Screen Shot 2024-01-27 at 01 53 48](https://github.com/flathub-infra/website/assets/1908896/7cb6829e-c5e3-4461-803e-b84501b6b426)

**Comment by razzeee** [comment]
> The vertical spacing between top bar and banners is a bit too high, I'd reduce it by a third or so

That's the "standard" and used on at least 8 pages right now

**Comment by razzeee** [comment]
After https://github.com/flathub-infra/website/pull/2615 the following is unaddressed

- [ ] The vertical spacing between top bar and banners is a bit too high, I'd reduce it by a third or so

Needs discussion and might need to be aligned on all pages


- [ ] The "App of the Day" and "Set up Flathub" cards should have the same height

I will work on this when I find time


- [ ] The card background is a bit too high contrast, I think on the mockup I'd layered a slight white-to-transparent gradient over it

Not sure I understand this or know what to do

**Comment by bertob** [comment]
>  The card background is a bit too high contrast, I think on the mockup I'd layered a slight white-to-transparent gradient over it

What I had on the mockup was two background layers: The logo pattern below, and above that a white-to-transparent gradient, so that the left side of the card has better legibility since that's where most of the text is.

![Screenshot from 2024-02-19 14-08-21](https://github.com/flathub-infra/website/assets/1908896/c401723f-01eb-49a1-989a-058a8ae8fc3d)

**Comment by razzeee** [comment]
I think I've had that, but did get rid of it for some reason, I can try to do it again and see where it fails

**Comment by bertob** [comment]
Playing with index-new I found that at the narrower widths where the screenshot is still visible the vertical offset looks a bit off, especially on larger screenshots:

![image](https://github.com/flathub-infra/website/assets/1908896/93bfa167-dd09-4d66-be36-d6464c709a55)

Perhaps there could be a breakpoint or something around 1300px or so adjusting the vertical offset for the screenshots?

**Comment by bertob** [comment]
Looking at the mockups, I think the general layout would benefit if we could use larger icons both on the main banner and for app of the day, it makes for a much more balanced layout with the screenshot

Mockup on the left, status quo on index-new on the right:

![banners-comparison](https://github.com/flathub-infra/website/assets/1908896/31760c37-7647-4dd3-b6b9-384d268a5fb5)

**Comment by razzeee** [comment]
@barthalion did you start generating 256px icons now? But I guess we can't rely on them being available in that size anyway.

**Comment by razzeee** [comment]
bigger icons seem to be a slightly bigger issue - as it needs backend changes, some heuristics and frontend changes. I'll close this for now, we should probably track the image changes somewhere else.
